include/rangers-api/Application/Player/GOCPlayerState.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+namespace app::player {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.